packages/core/src/interpreter.ts

```diff
@@ -296,7 +263,7 @@ export class Interpreter<
       | ((state: State<TContext, TEvent, any, TTypestate>) => void)
       | Observer<State<TContext, TEvent, any, TTypestate>>,
     // @ts-ignore
-    errorListener?: (error: any) => void,
+    errorListener?: (error: Error) => void,
```

Comment:
this is incorrect as currently errorListeners can be called with error events, but `Error` type represents regular javascript Error objects